packages/apps/job-launcher/server/src/modules/job/job.service.ts

```diff
@@ -287,15 +566,27 @@ export class JobService {
     await jobEntity.save();
 
     const escrowAddress = await escrowClient.createAndSetupEscrow(
-      NETWORKS[jobEntity.chainId as ChainId]!.hmtAddress,
-      [],
-      jobEntity.userId.toString(),
-      escrowConfig,
+        NETWORKS[jobEntity.chainId as ChainId]!.hmtAddress,
+        [
+          "0x40042D27Df0745e5e0cb619C0CfD7D51D59535DC",
+          "0x117AD648f29AFEc3aE313826DA3356dB1040ae4F",
+          "0xaE1FE9f8927AEEE885bbD4aFEFB02741B317902E",
+          "0x8b48d9908A80508A7Aa5900008Ab176987f418E2",
+          "0x3f12aF5134f44f2247aCca6c43b3ee6775abd914",
+          "0x26E7Ef2D05793c6D47c678f1F4B246856236F089",
+          "0xb56c770c2FA5222947a9e3C5Beb8dc46Dd656b5F",
+          "0x10c01D6B0396D9F3B2F06Fc5D3F60bc6Dc9cB1F6",
+          "0x93920Dbc3dcb192F1713d344Fb4Db6BD714A9ab9",
+          "0x727cB81C955e1De954473B5939Eda0dfDFe07281",
+          "0x6a13E0280740CC5bd35eeee33B470b5bBb93dF37"
+        ],
```

Comment:
what is this?

Reply:
These are trusted addresses for the testnet, I left them for testing. Deleted.